Object Containers/src/main/AgeSorted.java

```diff
@@ -0,0 +1,16 @@
+package main;
+
+import java.util.Comparator;
+
+public class AgeSorted implements Comparator<Person> {
+    /**
+     *  comparator for age in reverse order
+     * @param person
+     * @param t1
+     * @return
+     */
+    @Override
+    public int compare(Person person, Person t1) {
```

Comment:
Comparatorul ar fi trebuit sa ordoneze persoanele dupa varsta in ordine inversa, implementarea ta ordoneaza persoanele dupa varsta in ordine crescatoare.